packages/coinstac-ui/app/render/components/dashboard/dashboard.jsx

```diff
@@ -54,8 +54,8 @@ class Dashboard extends Component {
     const { router } = this.context;
 
     // Uncomment to clear local data
-    this.props.clearRuns();
-    this.props.initTestData();
+    // this.props.clearRuns();
```

Comment:
Be really great to have another way to do this not in the UI code

Reply:
Issue #405 is asking for a solution to this. I'm thinking maybe a button in the UI to reset? Could be in dev mode only.